TestRunner/TestRunner.csproj

```diff
@@ -33,6 +33,11 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="BHoM">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>C:\ProgramData\BHoM\Assemblies\BHoM.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
```

Comment:
Why was this needed? We didn't change anything except in Test oM in the end didn't we?

Reply:
It did not compile without it for me. Complained about a missing reference to `IObject`. And did not see any real harm in having BHoM referenced here, but might have missed something?